sdk/personalizer/ai-personalizer-rest/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure Personalizer client library for JavaScript
+
+Personalizer Client
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/personalizer/ai-personalizer-rest)
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/ai-personalizer)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/ai-personalizer?view=azure-node-preview)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
```

Comment:
I think we should also put the suggested npm version as well

Reply:
I will mimic the similar text in other projects.